pkg/metaserver/agent/metric/metric.go

```diff
@@ -134,6 +134,9 @@ func NewMalachiteMetricsFetcher(emitter metrics.MetricEmitter) MetricsFetcher {
 }
 
 type MalachiteMetricsFetcher struct {
+	cpuCgDataV1 cgroup.CPUCgDataV1
```

Comment:
if we store this here, why we still need metricStore? it's wired to just keep cpuCgDataV1/cpuCgDataV2 here, why not get previous Instructions from  metricStore?

Reply:
fixed